src/app/components/login/login.component.ts

```diff
@@ -33,28 +31,26 @@ export class LoginComponent implements OnInit {
     }
     return 'Not a valid password';
   }
- 
-  getLoginRejected(){
-    if (!this.authenticationService.getEmailChecked()){
-      console.log(this.authenticationService.getEmailChecked())
-      return 'Your email is incorrect'
-    }
-    if (!this.authenticationService.getPasswordChecked()){
-      console.log(this.authenticationService.getPasswordChecked())
-      return 'Your password is incorrect'
+
+  getLoginRejected() {
+    if (!this.authenticationService.getEmailChecked()) {
+      console.log(this.authenticationService.getEmailChecked());
+      return 'Your email is incorrect';
     }
-    else return 'Your email or password is incorrect'
+    if (!this.authenticationService.getPasswordChecked()) {
+      console.log(this.authenticationService.getPasswordChecked());
+      return 'Your password is incorrect';
+    } else return 'Your email or password is incorrect';
   }
```

Comment:
Not a good idea to let user know that username is incorrect or specifically password is incorrect, should be one error for both